<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-01T23:52:02 IST" name="Suite" finished-at="2025-03-01T23:52:56 IST" duration-ms="54313">
    <groups>
      <group name="sanity">
        <method signature="TC002_LoginWithCreds.login_with_valid_creds()[pri:0, instance:testCases.TC002_LoginWithCreds@6eb2384f]" name="login_with_valid_creds" class="testCases.TC002_LoginWithCreds"/>
      </group> <!-- sanity -->
      <group name="master">
        <method signature="TC002_LoginWithCreds.login_with_valid_creds()[pri:0, instance:testCases.TC002_LoginWithCreds@6eb2384f]" name="login_with_valid_creds" class="testCases.TC002_LoginWithCreds"/>
        <method signature="TC003_LoginwithDataProvider.login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" name="login_with_data_providers" class="testCases.TC003_LoginwithDataProvider"/>
      </group> <!-- master -->
    </groups>
    <test started-at="2025-03-01T23:52:02 IST" name="Test" finished-at="2025-03-01T23:52:56 IST" duration-ms="54313">
      <class name="testCases.TC003_LoginwithDataProvider">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:02 IST" name="setUp" finished-at="2025-03-01T23:52:08 IST" duration-ms="5753" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:10 IST" name="login_with_data_providers" data-provider="LoginData" finished-at="2025-03-01T23:52:15 IST" duration-ms="4643" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[vidyadevv@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vidya@999]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_data_providers -->
        <test-method signature="login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:15 IST" name="login_with_data_providers" data-provider="LoginData" finished-at="2025-03-01T23:52:17 IST" duration-ms="2179" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[amotoorisampletwo@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secondtwo]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_data_providers -->
        <test-method signature="login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:17 IST" name="login_with_data_providers" data-provider="LoginData" finished-at="2025-03-01T23:52:29 IST" duration-ms="11684" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[amotoorisamplethree@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secondthreee]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_data_providers -->
        <test-method signature="login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:29 IST" name="login_with_data_providers" data-provider="LoginData" finished-at="2025-03-01T23:52:30 IST" duration-ms="1495" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[amotoorisamplethree@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secondthree]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.TC003_LoginwithDataProvider.login_with_data_providers(TC003_LoginwithDataProvider.java:47)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_data_providers -->
        <test-method signature="login_with_data_providers(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:30 IST" name="login_with_data_providers" data-provider="LoginData" finished-at="2025-03-01T23:52:41 IST" duration-ms="10207" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[vidyadevv@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vidya@999]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at testCases.TC003_LoginwithDataProvider.login_with_data_providers(TC003_LoginwithDataProvider.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_data_providers -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC003_LoginwithDataProvider@676cf48]" started-at="2025-03-01T23:52:41 IST" name="tearDown" finished-at="2025-03-01T23:52:41 IST" duration-ms="439" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC003_LoginwithDataProvider -->
      <class name="testCases.TC002_LoginWithCreds">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC002_LoginWithCreds@6eb2384f]" started-at="2025-03-01T23:52:41 IST" name="setUp" finished-at="2025-03-01T23:52:52 IST" duration-ms="11271" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login_with_valid_creds()[pri:0, instance:testCases.TC002_LoginWithCreds@6eb2384f]" started-at="2025-03-01T23:52:52 IST" name="login_with_valid_creds" finished-at="2025-03-01T23:52:55 IST" duration-ms="2922" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_with_valid_creds -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC002_LoginWithCreds@6eb2384f]" started-at="2025-03-01T23:52:55 IST" name="tearDown" finished-at="2025-03-01T23:52:56 IST" duration-ms="701" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC002_LoginWithCreds -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
